cassandra_template.yaml

```diff
@@ -767,7 +767,7 @@ client_encryption_options:
 # can be:  all  - all traffic is compressed
 #          dc   - traffic between different datacenters is compressed
 #          none - nothing is compressed.
-internode_compression: all
+internode_compression: dc
```

Comment:
Cassandra is already quite CPU-intensive, hogging more CPU due to inter-DC traffic compression doesn't seem like a good default.